File: Challenges/Coding World Cup/2023/Day 28 - NZ vs SA - Secret Omen Patten.java
public class Solution {
    static int secretCode(int n, int[] a) {
        for (int i = 0; i < n - 2; i++) {
            if ((a[i] + a[i + 1] + a[i + 2]) % 10 == 0) {
                return 1;
            }
        }
        return 0;
    }
}

File: Challenges/Coding World Cup/2023/Day 28 - NZ vs SA - Secret Omen Patten.cpp
int secretCode(int n, vector<int> &a)
{
    // Write your code here.
    for (int i = 0; i <= n - 3; i++)
    {
        int x = a[i] % 10;
        int y = a[i + 1] % 10;
        int z = a[i + 2] % 10;
        if ((x + y + z) % 10 == 0)
            return 1;
    }

    return 0;
}
